docs/source/release/v6.3.0/33349_framework.rst

```diff
@@ -0,0 +1,10 @@
+Documentation
+-------------
+
+- The source links at the bottom of each algorithm page no longer include the
+  last-modified date. The dates were misleading in most cases as no meaningful
+  changes to the algorithm had actually occurred and could be something as
+  simple as formatting changes. Moving forward, each algorithm will now
+  document any significant changes of behaviour across versions in a dedicated
+  section within its own page.
```

Comment:
Would it be worth mentioning here that we want to call that dedicated section the Changelog?

Reply:
I think at this point that's probably too much detail. The contents on each page should make it obvious I think.